server/nav.ts

```diff
@@ -2,8 +2,9 @@ import { Request } from 'express';
 
 export const home = { title: 'Home', href: '/' };
 export const tutorSessions = { title: 'Tutor Sessions', href: '/tutor-sessions' };
+export const user = { title: 'Profile', href: '/user'}
 
-export const nav = [ home, tutorSessions ];
+export const nav = [ home, tutorSessions, user ];
```

Comment:
If you want this to be a "My Profile" kind of thing, I suggest that it would be to the right, near the login and "Welcome" part of the navbar. And I think for the href, it's going to have to link to `/users/<current user id>`